sjb/config/common/test_cases/origin_release.yml

```diff
@@ -6,6 +6,17 @@ extensions:
       title: "build an origin release"
       repository: "origin"
       script: |-
+        ORIGIN_TARGET_BRANCH=$(git rev-parse --abbrev-ref --symbolic-full-name HEAD)
```

Comment:
Sorry, I meant that there is a variable already available to you -- `$<REPO_NAME>_TARGET_BRANCH` for whatever is being tested. No need to do this.